subprojects/core/src/main/java/org/gradle/initialization/MixInLegacyTypesClassLoader.java

```diff
@@ -310,7 +310,8 @@ private void addStaticStringConstantGetters() {
                     constant.getKey(),
                     Type.getMethodDescriptor(Type.getType(String.class)), null, null);
                 mv.visitCode();
-                mv.visitLdcInsn(constant.getValue());
```

Comment:
I have a suggestion, since `constant.getValue()` isn't used any more, store the `name` of the field in it instead and use that below instead of removing `"get"`. Alternatively, convert it to a `Set` and just add `"get"` when needed.

Reply:
See 3a81bb9d8be543cadfe09a286816f6999ea2527a. WDYT?

Reply:
Yes, that's what I was thinking.